internal/cassandra/client.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/cpurta/tatanka/internal/model"
 )
 
+// Client cassandra_client methods abstraction
```

Comment:
Client is the interface that provides operations on a Cassandra cluster to read and write Trades, Periods, and Simulation results.